src/modules/webhooks/get-interaction/1.0/module.json

```diff
@@ -1,40 +1,1053 @@
+      "defaultValue": "personPhonePhoneNumber"
+    },
+    {
+      "name": "Person Device Id",
+      "value": "$.interaction.person.deviceId",
+      "defaultValue": "personDeviceId"
+    },
+    {
+      "name": "Person Person Id",
+      "value": "$.interaction.person.personId",
+      "defaultValue": "personPersonId"
+    },
+    {
+      "name": "Person Session Id",
+      "value": "$.interaction.person.sessionId",
+      "defaultValue": "personSessionId"
+    },
+    {
+      "name": "Person Phone Number",
+      "value": "$.interaction.person.phoneNumber",
+      "defaultValue": "personPhoneNumber"
+    },
+    {
+      "name": "User Id",
+      "value": "$.interaction.user.id",
+      "defaultValue": "userId"
+    },
+    {
+      "name": "User Channel",
+      "value": "$.interaction.user.channel",
+      "defaultValue": "userChannel"
+    },
+    {
+      "name": "User Live Trancription Transcribed",
+      "value": "$.interaction.user.liveTrancription.transcribed",
+      "defaultValue": "userLiveTrancriptionTranscribed"
+    },
+    {
+      "name": "User Live Trancription Active",
+      "value": "$.interaction.user.liveTrancription.active",
+      "defaultValue": "userLiveTrancriptionActive"
+    },
+    {
+      "name": "User Extension",
+      "value": "$.interaction.user.extension",
+      "defaultValue": "userExtension"
+    },
+    {
+      "name": "User Status Voip Status",
+      "value": "$.interaction.user.status.voipStatus",
+      "defaultValue": "userStatusVoipStatus"
+    },
+    {
+      "name": "User Status Status Type",
+      "value": "$.interaction.user.status.statusType",
+      "defaultValue": "userStatusStatusType"
+    },
+    {
+      "name": "User Status Status",
+      "value": "$.interaction.user.status.status",
+      "defaultValue": "userStatusStatus"
+    },
+    {
+      "name": "User Status Is A I D",
+      "value": "$.interaction.user.status.isAID",
+      "defaultValue": "userStatusIsAID"
+    },
+    {
+      "name": "User A I Dable",
+      "value": "$.interaction.user.AIDable",
+      "defaultValue": "userAIDable"
+    },
+    {
+      "name": "User Name Prefix",
+      "value": "$.interaction.user.namePrefix",
+      "defaultValue": "userNamePrefix"
+    },
+    {
+      "name": "User First Name",
+      "value": "$.interaction.user.firstName",
+      "defaultValue": "userFirstName"
+    },
+    {
+      "name": "User Last Name",
+      "value": "$.interaction.user.lastName",
+      "defaultValue": "userLastName"
+    },
+    {
+      "name": "User Title",
+      "value": "$.interaction.user.title",
+      "defaultValue": "userTitle"
+    },
+    {
+      "name": "User Email",
+      "value": "$.interaction.user.email",
+      "defaultValue": "userEmail"
+    },
+    {
+      "name": "User Full Name",
+      "value": "$.interaction.user.fullName",
+      "defaultValue": "userFullName"
+    },
+    {
+      "name": "User Attributes Time Tz",
```

Comment:
I think we can remove all the user attributes related vars as well as the user queues ones